Core/Src/bms_can_monitor.c

```diff
@@ -0,0 +1,51 @@
+#include "bms_can_monitor.h"
```

Comment:
nit: rename these files to just `bms.c` or `shepherd.c` or something. Should be a beginning of the BMS interface